packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -121,12 +122,25 @@ class ApplicationCommand extends Base {
       this.options ??= [];
     }
 
-    if ('default_permission' in data) {
+    if ('default_member_permissions' in data) {
       /**
-       * Whether the command is enabled by default when the app is added to a guild
-       * @type {boolean}
+       * The default bitfield used to determine whether this command be used in a guild
+       * @type {?Readonly<PermissionsBitField>}
        */
-      this.defaultPermission = data.default_permission;
+      this.defaultMemberPermissions = data.default_member_permissions
+        ? new PermissionsBitField(BigInt(data.default_member_permissions)).freeze()
+        : null;
+    } else {
+      this.defaultMemberPermissions ??= null;
+    }
+
+    if ('dm_permission' in data) {
+      /**
+       * Whether the command can be used in dms
+       * <info>This property is not present on guild commands</info>
+       * @type {boolean?}
```

Comment:
Does the `?` suffix work on our website?

Reply:
I don't believe this is a thing? I'm looking at https://jsdoc.app/tags-type.html

Reply:
gonna wait for decission above on this

Reply:
I think I'm in favor of being explicit here.

Reply:
I'll make it | null and add the default setter